plugins/woocommerce-blocks/tests/e2e/tests/product-gallery/product-gallery.block_theme.spec.ts

```diff
@@ -36,20 +36,34 @@ const test = base.extend< { pageObject: ProductGalleryPage } >( {
 	},
 } );
 
+/**
+ * Get the image id from the image element. Sometimes the image id is in the data-image-id attribute, sometimes in the id attribute.
```

Comment:
Do you know the reason why it's either this or that? I'm just curious, I don't think I caught it from a code elsewhere 🤔

Reply:
> id needs to be semantically unique so we can't have the large image with id="32" and then its corresponding thumbnail with the same id.

Ah right, makes sens! Thanks for explaining!

> I can update them to be streamlined and both use data-image-id I think that might be better to be honest.

Agree!

Reply:
Done in https://github.com/woocommerce/woocommerce/pull/55246/commits/7854ac6a434d60817f67e9de69b500a632f7ee03